doc/user/olm-catalog/generating-a-csv.md

```diff
@@ -11,40 +11,20 @@ This document describes how to manage the following lifecycle for your Operator
 Operator SDK projects have an expected [project layout][doc-project-layout]. In particular, a few manifests are expected to be present in the `deploy` directory:
 
 * Roles: `role.yaml`
+* ClusterRoles: `cluster_role.yaml`
 * Deployments: `operator.yaml`
 * Custom Resources (CR's): `crds/<full group>_<version>_<kind>_cr.yaml`
-* Custom Resource Definitions (CRD's): `crds/<full group>_<resource>_crd.yaml`.
+* CustomResourceDefinitions (CRD's): `crds/<full group>_<resource>_crd.yaml`.
 
-`gen-csv` reads these files and adds their data to a CSV in an alternate form.
-
-The following example config containing default values should be copied and written to `deploy/olm-catalog/csv-config.yaml`:
-
-```yaml
-crd-cr-paths:
-- deploy/crds
-operator-path: deploy/operator.yaml
-role-paths:
-- deploy/role.yaml
-```
-
-Explanation of all config fields:
-
-- `crd-cr-paths`: list of strings - a list of CRD and CR manifest file/directory paths. Defaults to `[deploy/crds]`.
-- `operator-path`: string - the operator `Deployment` manifest file path. Defaults to `deploy/operator.yaml`.
-- `role-paths`: list of strings - Role and ClusterRole manifest file paths. Defaults to `[deploy/role.yaml]`.
-- `operator-name`: string - the name used to create the CSV and manifest file names. Defaults to the project's name.
-
-**Note**: The [design doc][doc-csv-design] has outdated field information which should not be referenced.
-
-Fields in this config file can be modified to point towards alternate manifest locations, and passed to `gen-csv --csv-config=<path>` to configure CSV generation. For example, if I have one set of production CR/CRD manifests under `deploy/crds/production`, and a set of test manifests under `deploy/crds/test`, and I only want to include production manifests in my CSV, I can set `crd-cr-paths: [deploy/crds/production]`. `gen-csv` will then ignore `deploy/crds/test` when getting CR/CRD data.
+`gen-csv` extracts manifests from files in `deploy` that match the kinds above and adds them to the CSV. If any manifests are present that match one of the above kinds that you do _not_ want in your CSV, you can exclude them from the generator by setting `--exclude=[list of paths]`, ex. `--exlude="deploy/prod,deploy/test"`.
```

Comment:
Minor problem:

ex. `--exlude="deploy/prod,deploy/test"`.

-->

ex. `--exclude="deploy/prod,deploy/test"`.

[I cannot use the "suggestion" feature of Github in this PR, sorry]